Meta/Azure/Serenity.yml

```diff
@@ -93,12 +93,20 @@ jobs:
         SERENITY_KERNEL_CMDLINE: 'fbdev=off panic=shutdown system_mode=self-test'
         SERENITY_RUN: 'ci'
 
+    - task: PublishTestResults@2
+      inputs:
+        testResultsFormat: 'JUnit'
+        testResultsFiles: $(Build.SourcesDirectory)/Build/${{ parameters.arch }}clang/junit.xml
```

Comment:
what's up with the two mixed...interpolation(?)...styles here?
does `$(foo)` have some special meaning over `${{foo}}`?

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#understand-variable-syntax

Short answer: yes

${{foo}} is expanded at template instantiation time (since this is a job template, the real job is serenity/.azure-pipelines.yml) 

$(foo) is expanded at runtime. We don't know what runner version we've got and where it will choose to place our build directory.